### PR TITLE
Include more information for notifications about deleted comments

### PR DESCRIPTION
**Pull request by dmarcoux**
For deleted comments, we don't link to the project/package/request since the comment is gone. It also clearly states that the notification is for a deleted comment.

For all comments, we show in the notification's details who wrote the comment with the `from ABC` text.

example

### CODE REVIEW

**Comment by hennevogel** [comment]
The commenter is for sure the most important information. Especially as many people delete comments after reporting them...

**Comment by dmarcoux** [comment]
[Include more information about the report notifications for deleted reportables (M)](https://trello.com/c/u81KB9iX/2375-include-more-information-about-the-report-notifications-for-deleted-reportables-m)

File: src/api/app/components/notification_notifiable_link_component.rb [hunk 1]
```diff
@@ -69,6 +75,7 @@ def notifiable_link_text
   # rubocop:enable Metrics/CyclomaticComplexity
 
   # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/PerceivedComplexity
```
Comment (dmarcoux) [inline src/api/app/components/notification_notifiable_link_component.rb:78]: This method is already huge... and I'm adding more, but refactoring this method is out of scope for this PR.

**Review by eduardoj — APPROVED** [review]
